### PR TITLE
Ember 2.15 Release Blog Post

### PR DESCRIPTION
**Pull request by locks**
- [Rendered](https://github.com/emberjs/website/blob/2-15-release-blog-post/source/blog/2017-08-21-ember-2-15-released.md)
- [Preview](https://ember-website-pr-2948.herokuapp.com/blog/2017/08/21/ember-2-15-released.html)

## Sign-offs

- [x] Ember.js (@chancancode)
- [x] Ember Data (@bmac)
- [x] Ember CLI (@kellyselden @hjdivad)
- [x] Website (@locks, @mixonic)

## Checklist

- Ember
  - [x] Blog
  - [x] v2.15.0
  - [x] v2.16.0-beta.1
- Ember Data
  - [x] Blog
  - [x] v2.15.0
  - [ ] v2.16.0-beta.1
- Ember CLI
  - [x] Blog
  - [x] v2.15.0
  - [x] v2.16.0-beta.1
  - [x] v2.15.1 (Ember Data 2.15.0)
  - [x] v2.16.0-beta.2 (Ember Data 2.15.0)

See https://github.com/emberjs/website/pull/2824/files for inspiration.

### CODE REVIEW

**Comment by mixonic** [comment]
2.15 release post should mention end-of-life for 2.4-LTS

**Comment by rwjblue** [comment]
Just wanted to mention that we should make sure to include some verbiage about the router service phases (and likely create/update a meta issue that tracks implementing the rest of the functionality).

**Comment by locks** [comment]
ember cli updater?

**Comment by kellyselden** [comment]
@locks Are you asking what it is, or who's writing something for it? Because I was planning on writing a paragraph about it.

**Comment by locks** [comment]
@kellyselden just so we don't forget

**Comment by paulyoder** [comment]
Here's a small PR to remove an extraneous `a` from the post. https://github.com/emberjs/website/pull/2976

File: source/blog/2017-08-14-ember-2-15-released.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+export default Ember.Component.extend({
+  session: Ember.inject.service(),
+  title: 'The Curious Case'
+});
+```
+
+Would in 2.16 conventions be written as:
+
+```js
+import { inject } from '@ember/service';
```
Comment (Dhaulagiri) [inline source/blog/2017-08-14-ember-2-15-released.md]: There is some discussion about this [here](https://github.com/ember-cli/ember-modules-codemod/issues/34), but at the moment the codemod will translate `session: Ember.inject.service(),` into `import { inject as service } from '@ember/service';` which has to be used `session: service()`.

**Comment by mixonic** [comment]
Ok, I've updated the date on this to be tomorrow (Monday). All the sections are filled out and we're good to go.

I would really like to link to documentation for any of the API changes, but we're in a catch-22 until the documentation goes out.

File: source/blog/2017-08-21-ember-2-15-released.md [hunk 2]
```diff
@@ -86,16 +73,31 @@ export default Ember.Component.extend({
...
+Ember 2.15.0 implements
 [RFC #225](https://github.com/status200/rfcs/blob/ember-engines-mount-params/text/0000-ember-engines-mount-params.md) with some minor tweaks. The `{{mount}}`
 helper, used to invoke an Ember engine, now accepts the named argument of
-`model` to provide the model of the engine's application controller.
+`model` for an engine's application controller.
 
 For example, in an application you might pass some values from a template:
 
-```hbs
+```handlebars
```
Comment (locks) [inline source/blog/2017-08-21-ember-2-15-released.md]: does this template have a path?

Comment (mixonic) [inline source/blog/2017-08-21-ember-2-15-released.md]: No. It is an example of a template in an application, there is no specific location on disk that is relevant.

File: source/blog/2017-08-21-ember-2-15-released.md [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+Ember CLI will configure new apps to run tests in [headless Chrome](https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md)
+by default, instead of PhantomJS.
+
+PhantomJS has served the wider JavaScript community well for a long time, being
+a practical alternative to running browsers headless via tools like Xvbf. It
+has, however, been a proxy for what we really want to test– the browsers that
+users are running.
+
+Now that we can easily test in headless Chrome the motivation for using
+PhantomJS has diminished, and as a result it is [no longer actively maintained](https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md).
```
Comment (geoffreyd) [inline source/blog/2017-08-21-ember-2-15-released.md]: I don't think this link should be the same as the one above.
My guess is it should be pointing to something about phantomjs?

**Pull request merged**